frontend/src/app/search/search.component.html

```diff
@@ -224,55 +226,45 @@ <h4 class="mat-subheader">By Topic</h4>
               [fillColor]="hit.type.toLowerCase() === 'location' ? '#6C799C' : '#E57200'"
               fillOpacity="0.1"
               [clickable]="true"
-              [visible]="isInfoWindowOpen(hit.id)"
-              (circleClick)="showInfoWindow(hit.id)"
+              [visible]="isInfoWindowOpen() && selectedMapHit.id == hit.id"
+              (circleClick)="showInfoWindow(hit)"
               zIndex="-1"
             >
             </agm-circle>
             <agm-marker
-              *ngIf="hit.hasCoords()"
               [latitude]="hit.latitude + mapJitter(hit.id, true)"
               [longitude]="hit.longitude + mapJitter(hit.id, false)"
               [iconUrl]="{url: '/assets/map/' + hit.type + '-no-address.svg', anchor: {x: 24, y: 24}}"
-              (markerClick)="showInfoWindow(hit.id)"
+              (markerClick)="showInfoWindow(hit)"
             >
-              <agm-info-window [isOpen]="isInfoWindowOpen(hit.id)">
-                <a
-                  class="title mat-h2"
-                  href="/#/{{hit.type.toLowerCase()}}/{{hit.id}}"
-                >{{hit.title}}</a>
-                <p>{{hit.description}}</p>
-                <a
-                  class="chevron-link"
-                  [routerLink]="['/' + hit.type.toLowerCase() + '/' + hit.id]"
-                  routerLinkActive="router-link-active"
-                >{{hit.type.toLowerCase() === 'location' ? 'In-home therapy' : hit.type}} Details</a>
```

Comment:
We lose the "In-home therapy" clarification here by moving to one shared info window. Is this a trade-off that we need/want to make?